process_h5s.py

```diff
@@ -1,28 +1,45 @@
 #!/usr/bin/env python3
 
+import sys
 import argparse
 import csv
 import logging
 import pathlib
 import json
+import yaml
 import datetime
 
 import numpy as np
 import pandas as pd
 
+
 from epifor.gleam.simulation import SimSet, Simulation
 from epifor.regions import Regions
 from epifor.data.export import ExportDoc, ExportRegion
 
 log = logging.getLogger("process_data_hdf")
 
 
-def make_export_doc(regions, simset):
+def match_params_with_config(seasonality, air_traffic, config):
```

Comment:
You only need `config["scenarios"]` in this function, so I recommend to just send that in this function instead of the whole config (from the calling function: https://github.com/epidemics/epifor/pull/5/files#diff-9aa17cc5a6d46d58bb74ec086e1bb201R60).